### ISSUE TITLE
something wrong with preprocess_waymo

### ISSUE THREAD

**Issue by 451578928** [Pointcept/Pointcept#119]
I followed the README to preprocess the Waymo dataset, It was okay in the training and validation sets (truly generating the labels and velodyne data), but it didn't work in the testing set (only creating the velodyne folder but not generating the .bin files).

I modified the code in preprocess_waymo.py as follows:
```
...
data_group = tf.data.TFRecordDataset(file_path, compression_type="")
    for data in data_group:
        frame = open_dataset.Frame()
        frame.ParseFromString(bytearray(data.numpy()))
            if frame.lasers[0].ri_return1.segmentation_label_compressed:
                print("ok")
...
```
Only training and validation sets would output 'ok', but not the test set.
Would you be available to help me with this?

**Comment by Gofinge** [comment]
Hi, I also haven't validated the full workflow for the Waymo test set and submission. I will do this after finishing the release work of PTv3.

**Comment by yxchng** [comment]
any updates?

**Comment by Gofinge** [comment]
Hi, check the updates in v1.5.2
